lib/rexml/parsers/baseparser.rb

```diff
@@ -462,8 +460,7 @@ def pull_event
                   raise REXML::ParseException.new("Malformed CDATA: Missing end ']]>'", @source)
                 end
               end
-              raise REXML::ParseException.new( "Declarations can only occur "+
-                "in the doctype declaration.", @source)
+              raise REXML::ParseException.new("Malformed node", @source)
```

Comment:
Can we move this `raise` to the above `if`/`elsif`/`end`'s `else`?

Reply:
OK
I see.